docs/how-to-guides/software-management/software-support.md

```diff
@@ -49,4 +67,23 @@ Prioritization meetings are typically a loosely guided discussion in which the t
 1. Ask for any closing remarks
 
 ??? Note "Issue deferral"
-    Deferring an issue generally means "this is a good, important issue, but it requires further discussion." This discussion should not interrupt the flow of prioritization, and discussing deferred issues is usually a good final topic.
\ No newline at end of file
+    Deferring an issue generally means "this is a good, important issue, but it requires further discussion." This discussion should not interrupt the flow of prioritization, and discussing deferred issues is usually a good final topic.
+
+
+## Continous Support 
+
+An additional 2 weeks per quarter are dedicated for handling small issues. We refer to this as continious support, as it's suppport time between dedicated sprints. This amounts to roughly 6 hours a week to stay on top of GitHub notifications.  
+
+Examples of what is excpected to be worked during these continuous suport hours: 
```

Comment:
Whose exceptions? Who is providing all the transparency and context so an external person knows why their thing was ignored instead of another thing?

Reply:
Going to resolve this thread as it was moved, we can use the other thread to talk about updating this section.